src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java

```diff
@@ -480,6 +480,9 @@ protected void addTranslations()
 		addTooltip(MSItems.ROCK_COOKIE, "It's as edible as a rock");
 		addStore(MSItems.ROCK_COOKIE, "Grey-Colored Cookie");
 		addStoreTooltip(MSItems.ROCK_COOKIE, "This here is a true wonder-cookie! Totally! And they are even cheaper than regular cookies!");
+		add(MSItems.WOODEN_CARROT, "Wooden Carrot");
+		addTooltip(MSItems.WOODEN_CARROT, "It's as edible as a plank.");
+		addStoreTooltip(MSItems.WOODEN_CARROT, "You're not sure if this thing is edible but you sure as fuck don't want to try...... probably...");
```

Comment:
You probably would want to narrate the store tooltip as if it came from the consort selling it (not that the carrot is currently added to stores). Also, while we do have a few already in our lang file, I'd advice against using "fuck" or similar words.